cydarm.json

```diff
@@ -0,0 +1,1988 @@
+                    "order": 0,
+                    "data_type": "string",
+                    "required": true
+                },
+                "data": {
+                    "name": "data",
+                    "description": "Comment text",
+                    "order": 1,
+                    "data_type": "string",
+                    "required": true
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.case_uuid",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.acl",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.uuid",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "create case",
+            "identifier": "create_case",
+            "description": "Create case",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "acl": {
+                    "name": "acl",
+                    "description": "The ACL of the data object",
+                    "order": 0,
+                    "data_type": "string"
+                },
+                "assignee": {
+                    "name": "assignee",
+                    "description": "The name of the current assignee",
+                    "order": 1,
+                    "data_type": "string"
+                },
+                "closed": {
+                    "name": "closed",
+                    "description": "The time at which the case was closed",
+                    "order": 2,
+                    "data_type": "string"
+                },
+                "created": {
```

Comment:
Could this be generated in the backend? If not, it would be good to add the time format expected.

Reply:
If not specified, it is generated by the backend. I've added ISO-8601 in the description as expected format.